client/components/product-card/index.jsx

```diff
@@ -59,9 +59,11 @@ const ProductCard = ( {
 				</div>
 			</div>
 			<div className="product-card__description">
-				{ hasManageSubscriptionLink && (
+				{ purchase && (
 					<p>
-						<a href="/my-plan">{ translate( 'Manage Subscriptions' ) }</a>
+						<a href={ managePurchase( purchase.domain, purchase.id ) }>
+							{ translate( 'Manage Subscription' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **36** times:
`translate( 'Manage subscriptions' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/product-selector-purchase-object)

Reply:
Good suggestion, but it doesn't make sense for our use case.